dask-gateway/Dockerfile

```diff
@@ -34,7 +63,11 @@ WORKDIR /home/dask/
 
 # Install dask-gateway
 COPY --chown=dask:dask . /opt/dask-gateway
-RUN pip install --no-cache-dir \
+ARG PIP_CACHE_DIR=/tmp/pip-cache
```

Comment:
Does this need to be set a second time?

Reply:
Yepp, another stage, another built-time env var needed to configure pip where to store/read cache